src/ome_zarr_models/v04/image_label.py

```diff
@@ -46,23 +49,24 @@ class Source(Base):
     Source data for the labels.
     """
 
-    # TODO: add validation that this path resolves to something
-    image: str | None = "../../"
+    # TODO: add validation that this path resolves to a zarr image group
+    image: str | None = Field(
+        default="../../", description="Relative path to a Zarr group of a key image."
```

Comment:
Do we need a default value here?

Reply:
🤷 the spec says
> The default value is "../../" since most labels are stored under a subgroup named "labels/" (see above).

So I guess we should?

Reply:
yeah i guess so. weird language to have in a spec document but 🤷‍♂️